membershipextras.civix.php

```diff
@@ -443,3 +445,22 @@ function _membershipextras_civix_civicrm_alterSettingsFolders(&$metaDataFolders
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * (Delegated) Implements hook_civicrm_entityTypes().
+ *
+ * Find any *.entityType.php files, merge their content, and return.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_entityTypes
+ */
+
+function _membershipextras_civix_civicrm_entityTypes(&$entityTypes) {
```

Comment:
is this method auto-generated or you added it manually ?

Reply:
It's auto-generated by civix.